### PR TITLE
Better expose build dev project

### PR DESCRIPTION
**Pull request by liviur**
## Description

* Better positioning for development project so people won't miss it
* Removed slides - inserted screenshots inline
* Wrote better instructions including experimental iOS instructions for outsource developers

## Known issues

This is missing the Android instructions. @snooky23 please send them to me and I'll add them / add them with an extra commit

## Checklist

* [x] PR is scoped to one task
* [ ] Tests are included
* [x] Documentation is included

* [x] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

File: book/getting-started/download-development-project.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+### iOS
+
+#### Internal developers with access to Applicaster private repositories:
+1. Update the `Podfile` to include your `podspec` local path.
+2. Run `pod install --no-repo-update` in order to point to your local pod.
+3. Run `pod update YourPlugin --no-repo-update` to update your changes.
+
+#### External developers with no access to Applicaster private repositories (EXPERIMENTAL)
+We are currently working on simplifying these steps however the following steps represent a workaround that should work.
```
Comment (lomudi) [inline book/getting-started/download-development-project.md]: @liviur I don't think we have a point in talking about future developments or "workarounds", I would remove this line and will just jump to the steps.

**Comment by snooky23** [comment]
I recommend adding pictures that describe what you wrote in words.

File: book/getting-started/download-development-project.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+The downloaded projects are ready to run at this stage with no additional steps for debugging the initial version of the plugin.
+However sometimes it makes more sense to point to a local development version of the dependancy.
+
+The following instructions per platform will assist with changing your local project.
+
+### iOS
+
+#### Internal developers with access to Applicaster private repositories:
+1. Update the `Podfile` to include your `podspec` local path.
+2. Run `pod install --no-repo-update` in order to point to your local pod.
```
Comment (snooky23) [inline book/getting-started/download-development-project.md]: This should only be used if the developer wants to add a new pod to the list

File: book/getting-started/download-development-project.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+3. Run `pod update YourPlugin --no-repo-update` to update your changes.
```
Comment (snooky23) [inline book/getting-started/download-development-project.md]: this should be done if the developer wants to update a certain plugin in the podfile

**Comment by liviur** [comment]
> I recommend adding pictures that describe what you wrote in words.

@snooky23 please be specific

File: book/getting-started/download-development-project.md (hunk 2 above)
Comment (liviur) [reply book/getting-started/download-development-project.md]: or change where the plugin points to from Cocoapods to local

File: book/getting-started/download-development-project.md (hunk 1 above)
Comment (liviur) [reply book/getting-started/download-development-project.md]: removed

File: book/getting-started/download-development-project.md (hunk 3 above)
Comment (liviur) [reply book/getting-started/download-development-project.md]: fixed

**Comment by liviur** [comment]
@snooky23 please notice your action item here

**Comment by snooky23** [comment]
@lomudi - should I add the Android part before merge?

**Comment by lomudi** [comment]
@snooky23 i think it will be great.

**Pull request merged**